Funcky.Xunit.v3/Serializers/StringExtensions.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Funcky.Xunit.Serializers;
+
+internal static class StringExtensions
+{
+    public static Option<string> StripPrefix(this string s, string prefix)
+        => s.StartsWith(prefix)
+            ? s[prefix.Length..]
+            : Option<string>.None;
```

Comment:
the name of this method and what it does confuses me - you strip the prefix or return None, if there was no prefix to strip. Usually such a method returns simply the original value if the prefix can not be stripped - what the rationale here?

Reply:
Parsing: if the prefix is not found its an error or we need to check another case. I really like this function as getting your proposed solution is easily done by using `StripPrefix(s).GetOrElse(s)`.

Reply:
Btw I got the name from Rust's std :)